packages/flutter/lib/src/services/system_chrome.dart

```diff
@@ -113,6 +176,18 @@ class SystemUiOverlayStyle {
   /// When set to [Brightness.dark], the system navigation bar icons are dark.
   final Brightness? systemNavigationBarIconBrightness;
 
+  /// Overrides the contrast enforcement when setting a transparent navigation
```

Comment:
Does this only have an effect if systemNavigationBarColor is set to transparent? Maybe link the two properties in the docs?